build.zig

```diff
@@ -301,3 +351,59 @@ fn buildShaders(b: *Build) void {
         shdc_step.dependOn(&cmd.step);
     }
 }
+
+fn buildWasm(b: *Build, example: *CompileStep, options: ExampleOptions) !*RunStep {
+    // system path or package path
+    const emcc_path = b.findProgram(&.{"emcc"}, &.{}) catch b.pathJoin(&.{ options.packagePath(b).?, "upstream", "emscripten", "emcc" });
+    const emrun_path = b.findProgram(&.{"emrun"}, &.{}) catch b.pathJoin(&.{ options.packagePath(b).?, "upstream", "emscripten", "emrun" });
+
+    if (options.lib_sokol.rootModuleTarget().os.tag != .emscripten) {
+        std.log.err("Please build with 'zig build -Dtarget=wasm32-emscripten", .{});
+        return error.Wasm32EmscriptenExpected;
+    }
+
+    // call the emcc linker step as a 'system command' zig build step which
+    // depends on the libsokol and libgame build steps
+    try std.fs.cwd().makePath(b.fmt("{s}/web", .{b.install_path}));
+
+    var emcc_cmds = std.ArrayList([]const u8).init(b.allocator);
```

Comment:
Probably better to name this `emcc_cmd`, since it's just a single command (even though it's composed from multiple args). Sorry for nitpicking ;)